docs/contracts/functions.rst

```diff
@@ -70,18 +70,6 @@ with two integers, you would use something like the following::
 
 Function parameters can be used as any other local variable and they can also be assigned to.
 
-.. note::
-
-  An :ref:`external function<external-function-calls>` cannot accept a
-  multi-dimensional array as an input
-  parameter. This functionality is possible if you enable the ABI coder v2
-  by adding ``pragma abicoder v2;`` to your source file.
-
-  An :ref:`internal function<external-function-calls>` can accept a
-  multi-dimensional array without enabling the feature.
-
-.. index:: return array, return string, array, string, array of strings, dynamic array, variably sized array, return struct, struct
-
```

Comment:
Should this be deleted?

Reply:
I don't know these roles.

Reply:
I think it shouldn't. I think it adds extra links to the [Keyword Index](https://docs.soliditylang.org/en/latest/genindex.html) so that this section can be found through them.

In any case, it's not a part of the note so it should not be removed here.